cloud: new generation of TiDB Cloud support services - Connected Care (#19958)
pingcap/docs: tidb-cloud/connected-care-announcement.md

Reviewer: preview: https://pr.pingcap-docsite-preview.pages.dev/tidbcloud/connected-care-announcement

---
title: Announcing the New Generation of TiDB Cloud Support Services - Connected Care
summary: Introduces the new generation of TiDB Cloud support services - Connected Care.
---

# Announcing the New Generation of TiDB Cloud Support Services - Connected Care

As customers of all sizes continue to scale use cases and operations on TiDB Cloud, TiDB Cloud is committed to re-imagining its support services to accommodate their evolving needs. To deliver even greater value and seamless experiences, TiDB Cloud is excited to announce the launch of the next-generation support services, **Connected Care**, on **Feb 17, 2025**.

> As part of this transition, the current support plan offerings will no longer be available for purchase starting from **Feb 17, 2025**. However, TiDB Cloud will continue to provide full support to customers subscribing to legacy plans until their respective [retirement dates](#legacy-plans-replacement-and-retirement).

Reviewer: ```suggestion
As part of this transition, the current support plan offerings will no longer be available for purchase and will be regarded as legacy support plans starting from **Feb 17, 2025**. However, TiDB Cloud will continue to provide full support to customers subscribing to legacy plans until their respective [retirement dates](#legacy-plans-replacement-and-retirement).
```

> As part of this transition, the current support plan offerings will no longer be available for purchase starting from **Feb 17, 2025**. However, TiDB Cloud will continue to provide full support to customers subscribing to legacy plans until their respective [retirement dates](#legacy-plans-replacement-and-retirement).

Reviewer: ```suggestion
As part of this transition, the current support plan offerings will no longer be available for purchase and will be classified as legacy support plans starting from **Feb 17, 2025**. However, TiDB Cloud will continue to provide full support to customers subscribing to legacy plans until their respective [retirement dates](#legacy-plans-replacement-and-retirement).
```

| Connected: AI chat in IM                      | -                            | -                                             | ✔                                              | ✔                                         |
| Connected: IM subscription for TiDB Cloud alerts   | -                            | -                                             | ✔                                              | ✔                                         |
| Connected: IM subscription for ticket updates | -                            | -                                             | ✔                                              | ✔                                         |
| Connected: IM interaction for support tickets | -                            | -                                             | -                                              | ✔                                         |
| Technical account manager                     | -                            | -                                             | -                                              | ✔                                         |

> | Pricing                                       | Free                         | Greater of $29 or 10% of total monthly charges | Greater of $699 or 20% of total monthly charges | Contact sales                             |

Reviewer 2: Sorry for the late notice. Let's remove the detailed Pricing from the public doc. 
Here we need to delete this row. Thanks.

> | Pricing                                       | Free                         | Greater of $29 or 10% of total monthly charges | Greater of $699 or 20% of total monthly charges | Contact sales                             |

Reviewer: ```suggestion
```

> **Note**
>
> Customers from all four support plans have access to the [PingCAP support portal](https://tidb.support.pingcap.com/) for service requests.

## Differences between legacy support services and Connected Care support services

The support plans in the Connected Care services introduce a completely new set of features as follows:

- Connected: Clinic service

    This feature offers an advanced monitoring and diagnostic service, Clinic, designed to help you quickly identify performance issues, optimize your database, and enhance overall performance with detailed analysis and actionable insights.

- Connected: AI chat in IM

    This feature enables you to chat with an AI assistant through an instant message (IM) tool to receive immediate answers to your questions.

- Connected: IM subscription for TiDB Cloud alerts

>     This feature provides you with an easy way to subscribe to alert notifications via an instant message (IM) tool, keeping you informed about critical updates.

Reviewer: ```suggestion
    This feature provides you with an easy way to subscribe to alert notifications via an IM tool, keeping you informed about critical updates.
```

>     This feature provides you with an easy way to subscribe to alert notifications via an instant message (IM) tool, keeping you informed about critical updates.
> 
> - Connected: IM subscription for ticket updates
> 
>     This feature enables you to subscribe to your support ticket updates through an instant message (IM) tool.
> 
> - Connected: IM interaction for support tickets
> 
>     This feature enables you to quickly create and interact with support tickets through an instant message (IM) tool for streamlined communication.

Reviewer: ```suggestion
    This feature provides you with an easy way to subscribe to alert notifications via an instant message (IM) tool, keeping you informed about critical updates.

- Connected: IM subscription for ticket updates

    This feature enables you to subscribe to your support ticket updates through an IM tool.

- Connected: IM interaction for support tickets

    This feature enables you to quickly create and interact with support tickets through an IM tool for streamlined communication.
```

With these new features, the Connected Care services offer you better connectivity, more personalized support, and cost-effective solutions for different customer needs.

- New **Enterprise** and **Premium** plans: connect customers with modern communication tools and advanced AI capabilities through advanced monitoring service in Clinic, IM subscription for TiDB Cloud alerts, IM subscription for ticket updates, AI chat in IM, and IM interaction for support tickets.

> - New **Developer** plan: customers benefit from access to the same community and AI assistance as the **Basic** plan, while also enjoying a reduced subscription fee starting at $29 (decreased from $49 for the legacy **Standard** plan).

Reviewer: regarding "the same community and AI assistance", does AI mean TiDB.AI?  If yes, we need to make it clear to avoid users misunderstanding it with "Connected: AI chat in IM".

> - New **Developer** plan: customers benefit from access to the same community and AI assistance as the **Basic** plan, while also enjoying a reduced subscription fee starting at $29 (decreased from $49 for the legacy **Standard** plan).

Reviewer: ```suggestion
- New **Developer** plan: customers benefit from access to the same community and [tidb.ai](https://tidb.ai/) assistance as the **Basic** plan, while also enjoying a reduced subscription fee starting at $29 (decreased from $49 for the legacy **Standard** plan).
```

> - New **Developer** plan: customers benefit from access to the same community and [tidb.ai](https://tidb.ai/) assistance as the **Basic** plan, while also enjoying a reduced subscription fee starting at $29 (decreased from $49 for the legacy **Standard** plan).

Reviewer: ```suggestion
- New **Developer** plan: customers benefit from access to the same community and [tidb.ai](https://tidb.ai/) assistance as the **Basic** plan, while also enjoying a reduced subscription fee starting at $29 (decreased from $49 for the legacy **Standard** plan) and direct connections with unlimited access to technical support.
```

> - New **Developer** plan: customers benefit from access to the same community and [tidb.ai](https://tidb.ai/) assistance as the **Basic** plan, while also enjoying a reduced subscription fee starting at $29 (decreased from $49 for the legacy **Standard** plan) and direct connections with unlimited access to technical support.

Reviewer 2: Same - eliminate the pricing details:
while also enjoying a reduced subscription fee (decreased from the legacy **Standard** plan)

> - New **Developer** plan: customers benefit from access to the same community and [tidb.ai](https://tidb.ai/) assistance as the **Basic** plan, while also enjoying a reduced subscription fee starting at $29 (decreased from $49 for the legacy **Standard** plan) and direct connections with unlimited access to technical support.

Reviewer: ```suggestion
- New **Developer** plan: customers benefit from access to the same community and [tidb.ai](https://tidb.ai/) assistance as the **Basic** plan, while also enjoying direct connections with unlimited access to technical support.
```

> - New **Basic** plan: customers will be guided to join the active community channels, where they can engage with other community members and interact with an AI bot for technical assistance.

Reviewer: ```suggestion
- New **Basic** plan: customers will be guided to join the active community channels, where they can engage with other community members and interact with an [tidb.ai](https://tidb.ai/) for technical assistance.
```

---
title: Announcing the New Generation of TiDB Cloud Support Services - Connected Care
summary: Introduces the new generation of TiDB Cloud support services - Connected Care.
---

# Announcing the New Generation of TiDB Cloud Support Services - Connected Care

> As customers of all sizes continue to scale their use cases and operations on TiDB Cloud, TiDB Cloud is committed to re-imagine its support services to accommodate their evolving needs. To deliver even greater value and seamless experiences, TiDB Cloud is excited to announce the launch of the next-generation support services, **Connected Care**, on Feb 15, 2025.

Reviewer 3: ```suggestion
As customers of all sizes continue to scale their use cases and operations on TiDB Cloud, TiDB Cloud is committed to re-imagining its support services to accommodate their evolving needs. To deliver even greater value and seamless experiences, TiDB Cloud is excited to announce the launch of the next-generation support services, **Connected Care**, on Feb 15, 2025.
```

As part of this transition, the current support plan offerings will no longer be available for purchase starting from the launch date of the Connected Care services. However, TiDB Cloud will continue to provide full support to customers subscribing to legacy plans until their respective [retirement dates](#legacy-plans-replacement-and-retirement).

To ensure a smooth transition and access to the latest features, TiDB Cloud encourages customers to transit and adopt the Connected Care services.

## Connected Care

The new Connected Care support services are designed to bring the following benefits:

- Connect **Enterprise** and **Premium** plan customers with modern communication tools and advanced AI capabilities, delivering an enhanced and seamless customer experience.
- Connect **Basic** plan customers with the vibrant TiDB community, fostering closer engagement and interaction.

> There are four support plans: **Basic**, **Developer** (corresponding to the legacy **Standard** plan), Enterprise, and Premium. Customers from all four have access to the [PingCAP support portal](https://tidb.support.pingcap.com/) for service requests. Advanced features such as Technical Account Managers and [Placeholder] are also available.

Reviewer 3: ```suggestion
There are four support plans: **Basic**, **Developer** (corresponding to the legacy **Standard** plan), **Enterprise**, and **Premium**. Customers from all four have access to the [PingCAP support portal](https://tidb.support.pingcap.com/) for service requests. Advanced features such as Technical Account Managers and [Placeholder] are also available.
```

> The following is an overview of Connected Care services. See the [full details here](/tidb-cloud/connected-care-detail.md).

Reviewer 3: ```suggestion
The following is an overview of Connected Care services. For more information, see [Connected Care Details](/tidb-cloud/connected-care-detail.md).
```

| Connected: IM subscription for Cloud alerts   |                              |                                               | Y                                              | Y                                         |
| Connected: IM subscription for ticket updates |                              |                                               | Y                                              | Y                                         |
| Connected: IM interaction for support tickets |                              |                                               |                                                | Y                                         |
| Technical account manager                     |                              |                                               |                                                | Y                                         |
| Pricing                                       | Free                         | Greater of $29 or 10% of total monthly charges | Greater of $699 or 20% of total monthly charges | Contact sales                             |

> ## Legacy Plans replacement and retirement

Reviewer 3: ```suggestion
## Replacement and retirement of legacy plans 
```

In the new Connected Care services, although **Basic**, **Enterprise**, and **Premium** support plans use the same names as those in legacy plans, they refer to different plans with different service commitments.

> Once a legacy support plan shuts down, TiDB Cloud will no longer support it. If you does transition to any support plan in Connected Care by the relevant shutdown date, you will be automatically transitioned to the **Basic** support plan in Connected Care.

Reviewer 3: ```suggestion
Once a legacy support plan shuts down, TiDB Cloud will no longer support it. If you do transition to any support plan in Connected Care by the relevant shutdown date, you will be automatically transitioned to the **Basic** support plan in Connected Care.
```

The following table lists the shutdown schedules for the legacy support plans:

| Support plans                        | Shutdown date |
|:----------------------------------------|:--------------|
| Legacy **Basic** plan                     | Feb 15, 2025  |
| Legacy **Standard** plan                           | Feb 15, 2025  |
| Legacy **Enterprise** and **Premium** plans | Jan 15, 2026  |

## FAQs

### What's the difference between the legacy services and the new Connected Care services?

The support plans in the new Connected Care services provide better connectivity, more personalized support, and cost-effective solutions for different customer needs.

> - New **Enterprise** and **Premium** plans: A completely new set of features designed to help customers stay more 'Connected' is now available. These features include Clinic advanced monitoring service, IM subscription for Cloud alerts, IM subscription for ticket updates, AI chat in IM, and IM interaction for support tickets.

Reviewer 3: ```suggestion
- New **Enterprise** and **Premium** plans: a completely new set of features designed to keep customers more 'Connected' is now available. These features include advanced monitoring service with Clinic, IM subscription for Cloud alerts, IM subscription for ticket updates, AI chat in IM, and IM interaction for support tickets.
```

> - **Developer** plan: customers benefit from access to the same community and AI assistance as the **Basic** plan, while also enjoying a reduced subscription fee starting at $29 (decreased from $49 for the legacy **Standard** service)

Reviewer 3: ```suggestion
- **Developer** plan: customers benefit from access to the same community and AI assistance as the **Basic** plan, while also enjoying a reduced subscription fee starting at $29 (decreased from $49 for the legacy **Standard** service).
```

---
title: Announcing the new generation of TiDB Cloud support services - Connected Care
summary: Announcing the new generation of TiDB Cloud support services - Connected Care
---

# Announcing the new generation of TiDB Cloud support services - Connected Care

## Overview

As customers of all sizes continue to scale their use cases and operations on TiDB Cloud, TiDB Cloud is committed to re-imagine its support services to accommodate their evolving needs. To deliver even greater value and seamless experiences, TiDB Cloud is excited to announce the launch of the next-generation support services 'Connected Care' on Feb 15, 2025.

The current support offerings will no longer be available for purchase as of the launch date of the Connected Care services. However, TiDB Cloud will continue to provide full support to customers subscribing to legacy plans until their respective retirement dates.

To ensure a smooth transition and access to the latest features, TiDB Cloud encourages customers to transit and adopt the Connected Care services.

## Connected Care

The new Connected Care support services are designed in order to:

> - Connect Enterprise and Premium plan customers with modern communication tools and advanced AI capabilities, delivering an enhanced and seamless customer experience
> - Connect Basic plan customers with the vibrant TiDB community, fostering closer engagement and interaction

Reviewer: ```suggestion
- Connect **Enterprise** and **Premium** plan customers with modern communication tools and advanced AI capabilities, delivering an enhanced and seamless customer experience.
- Connect **Basic** plan customers with the vibrant TiDB community, fostering closer engagement and interaction.
```

> There are four layers of support plans: Basic, Developer (corresponds to the 'Standard' tier in the legacy plan), Enterprise, and Premium. Customers from all four have access to the PingCAP support portal for service requests. Advanced features such as Technical Account Managers and [Placeholder] are also available.

Reviewer: [Placeholder]？

> There are four layers of support plans: Basic, Developer (corresponds to the 'Standard' tier in the legacy plan), Enterprise, and Premium. Customers from all four have access to the PingCAP support portal for service requests. Advanced features such as Technical Account Managers and [Placeholder] are also available.

Reviewer: [Placeholder]？

---
title: Announcing the New Generation of TiDB Cloud Support Services - Connected Care
summary: Introduces the new generation of TiDB Cloud support services - Connected Care.
---

> # Announcing the New Generation of TiDB Cloud Support Services - Connected Care

Reviewer 3: Should we use the verb Announce instead of Announcing for the title?

> As customers of all sizes continue to scale use cases and operations on TiDB Cloud, TiDB Cloud is committed to re-imagining its support services to accommodate their evolving needs. To deliver even greater value and seamless experiences, TiDB Cloud is excited to announce the launch of the next-generation support services, **Connected Care**, on **Feb 15, 2025**.

Author: ```suggestion
As customers of all sizes continue to scale use cases and operations on TiDB Cloud, TiDB Cloud is committed to re-imagining its support services to accommodate their evolving needs. To deliver even greater value and seamless experiences, TiDB Cloud is excited to announce the launch of the next-generation support services, **Connected Care**, on **Feb 17, 2025**.
```

> As part of this transition, the current support plan offerings will no longer be available for purchase starting from **Feb 15, 2025**. However, TiDB Cloud will continue to provide full support to customers subscribing to legacy plans until their respective [retirement dates](#legacy-plans-replacement-and-retirement).

Author: ```suggestion
As part of this transition, the current support plan offerings will no longer be available for purchase starting from **Feb 17, 2025**. However, TiDB Cloud will continue to provide full support to customers subscribing to legacy plans until their respective [retirement dates](#legacy-plans-replacement-and-retirement).
```

> To ensure a smooth transition and access to the latest features, TiDB Cloud encourages customers to transit and adopt the Connected Care services.

Reviewer 3: L64 users `transition`.
```suggestion
To ensure a smooth transition and access to the latest features, TiDB Cloud encourages customers to transition and adopt the Connected Care services.
```

## Connected Care

The Connected Care services are designed to strengthen your connection with TiDB Cloud through modern communication tools, proactive support, and advanced AI capabilities, delivering a seamless and customer-centric experience.

> In the Connected Care services, there are four support plans: **Basic**, **Developer** (corresponding to the legacy **Standard** plan), **Enterprise**, and **Premium**.

Reviewer: ```suggestion
In the Connected Care services, there are four support plans: **Basic**, **Developer** (corresponding to the legacy **Standard** plan), **Enterprise**, and **Premium**. Advanced features such as technical account managers, instant messenger integration support, and Clinic services (an advanced monitoring and diagnostic service) are also available in these plans.
```

| Billing and account inquiries                 | ✔                            | ✔                                             | ✔                                              | ✔                                         |
| Technical support                             | -                            | ✔                                             | ✔                                              | ✔                                         |
| Initial response time                         | -                            | Business hours                                | 7x24                                           | 7x24                                      |
| Connected: Clinic service                     | -                            | -                                             | ✔                                              | ✔                                         |
| Connected: AI chat in IM                      | -                            | -                                             | ✔                                              | ✔                                         |

> | Connected: IM subscription for Cloud alerts   | -                            | -                                             | ✔                                              | ✔                                         |

Reviewer: ```suggestion
| Connected: IM subscription for TiDB Cloud alerts   | -                            | -                                             | ✔                                              | ✔                                         |
```

| Connected: IM subscription for ticket updates | -                            | -                                             | ✔                                              | ✔                                         |
| Connected: IM interaction for support tickets | -                            | -                                             | -                                              | ✔                                         |
| Technical account manager                     | -                            | -                                             | -                                              | ✔                                         |
| Pricing                                       | Free                         | Greater of $29 or 10% of total monthly charges | Greater of $699 or 20% of total monthly charges | Contact sales                             |

> **Note**
>

> > Customers from all four support plans have access to the [PingCAP support portal](https://tidb.support.pingcap.com/) for service requests. Advanced features such as Technical Account Managers and [Placeholder] are also available.

Reviewer: @iguoyr Please confirm the [Placeholder] above. Thanks.

Reviewer 4: > @iguoyr Please confirm the [Placeholder] above. Thanks.

Advanced features such as technical account managers, instant messenger integration support, and Clinic services are also available.

> > Customers from all four support plans have access to the [PingCAP support portal](https://tidb.support.pingcap.com/) for service requests. Advanced features such as Technical Account Managers and [Placeholder] are also available.

Reviewer: ```suggestion
> Customers from all four support plans have access to the [PingCAP support portal](https://tidb.support.pingcap.com/) for service requests.
```

## Differences between legacy support services and Connected Care support services

The support plans in the new Connected Care services offer you better connectivity, more personalized support, and cost-effective solutions for different customer needs.

> - New **Enterprise** and **Premium** plans: connect customers with modern communication tools and advanced AI capabilities through a completely new set of features, such as advanced monitoring service in Clinic, IM subscription for TiDB Cloud alerts, IM subscription for ticket updates, AI chat in IM, and IM interaction for support tickets.

Reviewer 3: ```suggestion
- New **Enterprise** and **Premium** plans: connect customers with modern communication tools and advanced AI capabilities through a completely new set of features, such as advanced monitoring service in [TiDB Cloud Clinic](tidb-cloud/tidb-cloud-clinic.md), IM subscription for TiDB Cloud alerts, IM subscription for ticket updates, AI chat in IM, and IM interaction for support tickets.
```

Reviewer: tidb-cloud/tidb-cloud-clinic.md will be available on Feb 17, 2025, so we are not able to provide the link now.

> - New **Developer** plan: customers benefit from access to the same community and AI assistance as the **Basic** plan, while also enjoying a reduced subscription fee starting at $29 (decreased from $49 for the legacy **Standard** plan)

Reviewer: ```suggestion
- New **Developer** plan: customers benefit from access to the same community and AI assistance as the **Basic** plan, while also enjoying a reduced subscription fee starting at $29 (decreased from $49 for the legacy **Standard** plan).
```

- New **Basic** plan: customers will be guided to join the active community channels, where they can engage with other community members and interact with an AI bot for technical assistance.

## Replacement and retirement of legacy support plans

The following table lists the shutdown schedules for the legacy support plans:

| Support plans                        | Shutdown date |
|:----------------------------------------|:--------------|

> | Legacy **Basic** plan                     | Feb 15, 2025  |

Author: ```suggestion
| Legacy **Basic** plan                     | Feb 17, 2025  |
```

> | Legacy **Standard** plan                           | Feb 15, 2025  |

Author: ```suggestion
| Legacy **Standard** plan                           | Feb 17, 2025  |
```

| Legacy **Enterprise** and **Premium** plans | Jan 15, 2026  |

> Once a legacy support plan shuts down, TiDB Cloud will no longer support it. If you does not transition to any support plan in Connected Care by the relevant shutdown date, you will be automatically transitioned to the **Basic** support plan in Connected Care.

Reviewer: ```suggestion
Once a legacy support plan shuts down, TiDB Cloud will no longer support it. If you do not transition to any support plan in Connected Care by the relevant shutdown date, you will be automatically transitioned to the **Basic** support plan in Connected Care.
```

---
title: Announcing the New Generation of TiDB Cloud Support Services - Connected Care
summary: Introduces the new generation of TiDB Cloud support services - Connected Care.
---

# Announcing the New Generation of TiDB Cloud Support Services - Connected Care

As customers of all sizes continue to scale use cases and operations on TiDB Cloud, TiDB Cloud is committed to re-imagining its support services to accommodate their evolving needs. To deliver even greater value and seamless experiences, TiDB Cloud is excited to announce the launch of the next-generation support services, **Connected Care**, on **Feb 17, 2025**.

> As part of this transition, the current support plan offerings will no longer be available for purchase and will be classified as legacy support plans starting from **Feb 17, 2025**. However, TiDB Cloud will continue to provide full support to customers subscribing to legacy plans until their respective [retirement dates](#legacy-plans-replacement-and-retirement).

Reviewer: ```suggestion
As part of this transition, the current support plan offerings will no longer be available for purchase and will be classified as legacy support plans starting from **Feb 17, 2025**. However, TiDB Cloud will continue to provide full support to customers subscribing to legacy plans until their respective [retirement dates](#replacement-and-retirement-of-legacy-support-plans).
```

To ensure a smooth transition and access to the latest features, TiDB Cloud encourages customers to transition and adopt the Connected Care services.

## Connected Care

The Connected Care services are designed to strengthen your connection with TiDB Cloud through modern communication tools, proactive support, and advanced AI capabilities, delivering a seamless and customer-centric experience.

In the Connected Care services, there are four support plans: **Basic**, **Developer** (corresponding to the legacy **Standard** plan), **Enterprise**, and **Premium**.

> **Note**
>
> Although **Basic**, **Enterprise**, and **Premium** support plans use the same plan names as those in legacy plans, they refer to different plans with different service commitments.

> The following table provides an overview for each support plan in the Connected Care services. For more information, see the [Connected Care Details](/tidb-cloud/connected-care-detail.md).

Reviewer: ```suggestion
The following table provides an overview for each support plan in the Connected Care services. For more information, see [Connected Care Details](/tidb-cloud/connected-care-detail.md).
```
